src/css/docs.css

```diff
@@ -188,8 +208,21 @@ table tr td + td + td:last-child {
   }
 }
 
-  /* Tons of hacks for the algolia search to look good on mobile */
+/* Tons of hacks for the algolia search to look good on mobile */
 @media only screen and (max-width: 600px) {
+  .DocSearch.DocSearch-Button {
+    width: 100%;
+    margin: 0;
+    height: 2.25rem;
+    font-size: 1rem;
+    border-radius: 0;
+    color: #ffffff;
+    background-color: #9a2829;
+    font-family: var(--ifm-font-family-base);
+  }
+  span.DocSearch-Button-Placeholder {
+    display: block;
+  }
```

Comment:
Same as above. Also, you added code instead of replacing the navbar__search stuff,